.scalafmt.conf

```diff
@@ -1,8 +1,7 @@
-version = "2.4.2"
+version = "2.5.2"
 project.git = true
-align = none
+align.preset=none
 docstrings = javadoc
-danglingParentheses = true
```

Comment:
As far as i can tell, the multiple settings of `danglingParentheses` all default to `true`.

https://scalameta.org/scalafmt/docs/configuration.html#danglingparentheses